src/alpine/3.21/helix/Dockerfile

```diff
@@ -0,0 +1,60 @@
+FROM library/alpine:3.21 AS venv
+
+RUN apk add --upgrade --no-cache \
+        cargo \
+        python3-dev \
+        build-base \
+        libffi-dev \
+        openssl-dev \
+        gcc \
+        linux-headers
+
+RUN python3 -m venv /venv && \
+        source /venv/bin/activate && \
+        pip download --no-deps helix-scripts --index-url https://dnceng.pkgs.visualstudio.com/public/_packaging/helix-client-prod/pypi/simple && \
+        pip install ./helix_scripts-*-py3-none-any.whl
+
+FROM library/alpine:3.21
+
+# Install .NET and test dependencies
+RUN apk add --upgrade --no-cache \
+        bash \
+        coreutils \
+        curl \
+        icu-data-full \
+        icu-libs \
+        iputils \
+        krb5-libs \
+        lldb \
+        llvm \
+        lttng-ust \
+        musl-locales \
+        numactl \
+        openssl \
+        python3 \
+        python3-dev \
+        py3-pip \
+        sudo \
+        tzdata
+
+# Install libmsquic from testing repository
+RUN apk add --repository=http://dl-cdn.alpinelinux.org/alpine/edge/community/ libmsquic
```

Comment:
We don't need to pull `libmsquic` from testing repository anymore on alpine starting with version 3.21, we should able to pull it directly as

`apk add libmsquic`

Reply:
I had noticed that and was wondering if this would be the guidance. Cool!

I'll make a PR for that.

Reply:
https://github.com/dotnet/dotnet-buildtools-prereqs-docker/pull/1292